src/Parser/Core/Modules/AbilityTracker.js

```diff
@@ -14,6 +14,7 @@ class AbilityTracker extends Module {
     event.manaCost = this.getManaCost(event);
     event.rawManaCost = this.getRawManaCost(event);
     event.hasInnervate = this.owner.selectedCombatant.hasBuff(SPELLS.INNERVATE.id, event.timestamp);
+    event.hasSymbolOfHope = this.owner.selectedCombatant.hasBuff(SPELLS.SYMBOL_OF_HOPE_TALENT.id, event.timestamp);
```

Comment:
Hmmm Symbol of Hope and Innervate both have the same effect, so perhaps they should be merged into 1 prop. I don't like the idea of adding more and more props to this with all the same effect.

`event.isManaCostNullified` perhaps?